.github/workflows/binaries.yml

```diff
@@ -19,7 +19,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@stable
         with:
           profile: minimal
```

Comment:
This one is fine, but we don't need the `toolchain` and`override` key-value pairs here